config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
+            or text ‘ACTION’ to 81025
+          show_to_nations:
+          - Northern Ireland
       get_food:
-        title: "If you’re unable to get food"
+        title: If you’re unable to get food
         show_options:
-          - "No"
-          - "Not sure"
+        - 'No'
+        - Not sure
         items:
-          - id: "0112"
-            text: "Find out if you can get support as a clinically extremely vulnerable person."
-            href: "https://www.gov.uk/coronavirus-extremely-vulnerable/"
-            show_to_nations:
-              - England
-            show_to_vulnerable_person: true
-          - id: "0038"
-            text: "Find out if you can get help if you’re clinically at high risk from coronavirus (Gov.scot)."
-            href: "https://www.gov.scot/publications/covid-shielding/pages/overview/"
-            show_to_nations:
-              - Scotland
-          - id: "0039"
-            text: "You might be able to phone or email your local shops to get a food delivery, or get food online."
-          - id: "0040"
-            text: 'If you can, ask friends, family or neighbours to help you get food. <a href="https://www.gov.uk/guidance/coronavirus-covid-19-accessing-food-and-essential-supplies" class="govuk-link">Find out how you can pay for your shopping safely.</a>'
-          - id: "0041"
-            text: 'If there’s no one to help you get food, prescriptions, or other essentials, find out if you can get help from a volunteer through the <a href="https://volunteering.royalvoluntaryservice.org.uk/nhs-volunteer-responders-portal/isolating" class="govuk-link"> NHS Volunteer Responders programme.</a>'
-            show_to_nations: 
-              - England
-              - Wales
-              - Northern Ireland
-          - id: "0042"
-            text: 'If there’s no one to help you get food, prescriptions, or other essentials, <a href="https://www.gov.scot/publications/coronavirus-covid-19-help-for-vulnerable-people/" class="govuk-link"> call the coronavirus helpline for Scotland </a> or find out if you can get help from a volunteer through the <a href="https://volunteering.royalvoluntaryservice.org.uk/nhs-volunteer-responders-portal/isolating" class="govuk-link"> NHS Volunteer Responders programme.</a>'
-            show_to_nations: 
-              - Scotland
-          - id: "0043"
-            text: 'If you need urgent help and have no other support, <a href="https://www.gov.uk/coronavirus-local-help" class="govuk-link">contact your local council</a> to find out what services are available in your area.'
-          - id: "0044"
-            text: "Get more information on accessing food and other essential supplies."
-            href: "https://www.gov.uk/guidance/coronavirus-covid-19-accessing-food-and-essential-supplies"
+        - id: '0112'
+          text: Find out if you can get support as a clinically extremely vulnerable
+            person.
```

Comment:
```suggestion
          text: |
            Find out if you can get support as a clinically extremely vulnerable
            person.
```

Reply:
See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216